### ISSUE TITLE
only global Activities showing up - version 1.13 AND 1.15.2

### ISSUE THREAD

**Issue by yroberttissot** [kevinpapst/kimai2#2814]
**Describe the bug**
when I create project related Activities (with active projects), these activities never appear on the timesheet creation page (Even after selecting the concerned project).

The only way to create timesheets with that newly created activity is with the version 1.15.2 by opening the activity and clicking play. Then it is properly selected with the project.

**To Reproduce**
1. Create a new activity
2. Link it to an active customer and project
3. Save
4. Try to record time with that project and activity. The activity won't be visible

**Logfile**
```
[2021-10-05 08:46:40] php.CRITICAL: Uncaught Error: Call to undefined method ReflectionProperty::getType() {"exception":"[object] (Error(code: 0): Call to undefined method ReflectionProperty::getType() at /home/clients/d3ba84e1add40dfe16a751e5ab76e7c7/sites/kimai.inoshift.com/vendor/jms/serializer/src/Metadata/Driver/TypedPropertiesDriver.php:107)"} []
[2021-10-05 08:46:40] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\UndefinedMethodError: "Attempted to call an undefined method named "getType" of class "ReflectionProperty"." at /home/clients/d3ba84e1add40dfe16a751e5ab76e7c7/sites/kimai.inoshift.com/vendor/jms/serializer/src/Metadata/Driver/TypedPropertiesDriver.php line 107 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\UndefinedMethodError(code: 0): Attempted to call an undefined method named \"getType\" of class \"ReflectionProperty\". at /home/clients/d3ba84e1add40dfe16a751e5ab76e7c7/sites/kimai.inoshift.com/vendor/jms/serializer/src/Metadata/Driver/TypedPropertiesDriver.php:107)"} []
[2021-10-05 08:46:40] security.DEBUG: Stored the security token in the session. {"key":"_security_secured_area"} []
[2021-10-05 08:46:41] request.INFO: Matched route "get_activities". {"route":"get_activities","route_parameters":{"_route":"get_activities","_controller":"App\\API\\ActivityController::cgetAction","_format":"json"},"request_uri":"https://kimai.inoshift.com/public/api/activities?project=10&visible=1","method":"GET"} []
```

**Additional context**
 - Kimai version: 1.15.2
 - PHP version: 7.4.24
- Server: Apache server
 - User Device: Windows Desktop
 - Browser Firefox

**Comment by yroberttissot** [comment]
well, I had the same issue before updating. On version 1.13
i followed the steps https://www.kimai.org/documentation/updates.html (without changing permissions)

**Comment by kevinpapst** [comment]
Please open Doctor and check PHP version. 
You are either not using 7.4 or you are using 7.4 in web context but a different one in console context.
AFAIR the error `Attempted to call an undefined method named "getType" of class "ReflectionProperty".` indicates a wrong PHP version, but I think the underlying issue is an outdated composer package.

**Comment by yroberttissot** [comment]
mh indeed, in Doctor it indicates 7.3.31
with the command php --version i get 7.4.24
how is that possible?

**Comment by kevinpapst** [comment]
Well, command line and web-engine are two different binaries and you can set it in your environment which one to use.
But don't ask how, every system treats it differently.
So command line did install a newer package which requires 7.4 and this package fails under the 7.3 web-context, because a method is missing. 
Easiest fix: upgrade your web-context to 7.4 as well.
Second option: Downgrade your command line to 7.3 and run composer and cache reload again.

**Comment by yroberttissot** [comment]
all solved by updating the PHP version. thanks!
